ConservationInternational/resilienceatlas
# Feature/RA2-237-timeline

Temporal layers

- [x] Display datepicker
- [x] Add date in URL
- [x] Check that COG layers are correctly displayed
- [x] Check that Carto layers are correctly displayed
- [x] Check that analysis are displayed
- [x] Check that interaction is working


## Testing instructions

Test the following layers on the Map:
- Map test layers > test cog [temporal]

- Map test layers > Test temporal layer (cartodb)

You should see a datepicker on the legend. After changing it the URL should change and the layer should rerender to see the correct data

## Tracking

https://vizzuality.atlassian.net/browse/RA2-237?atlOrigin=eyJpIjoiODUwOTYxYmFmOWRhNDAxZWI5YTdjNmFlY2QwMzYyMDYiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/RA2-237?atlOrigin=eyJpIjoiMjFjMTU0NjczZmI5NGNlMjk0MzEzMjViYmVjMTU1MWQiLCJwIjoiaiJ9
